downstream/modules/platform/proc-create-controller-form-view.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc_create-controller-form-view_{context}"]
+
+= Create-{ControllerName}-form-view
+
+== Using the Form View
+
+. Ensure *Form view* is selected. It should be selected by default.
+. Enter the name of the new controller.
+. Add any labels necessary.
```

Comment:
This part seems vague. Are there labels that should be there? Seems like some guidance is needed here.

Reply:
Labels are not necessary.  Labels can be added to make it easier to filter for objects in your Openshift cluster.  For example, you could add a label like:

![image](https://user-images.githubusercontent.com/11698892/193658460-15fa0483-8906-4c3b-801d-ba14f2b33f5d.png)